estate/models/estate_property.py

```diff
@@ -0,0 +1,111 @@
+from odoo import models, fields, api
+from dateutil.relativedelta import relativedelta
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "estate.property"
+
+    name = fields.Char(string="Title", required=True)
+    description = fields.Text(string="Description")
+    postcode = fields.Char(string="Postcode")
+    date_availability = fields.Date(
+        string="Availability From",
+        copy=False,
+        default=lambda self: fields.Date.today() + relativedelta(months=3),
+    )
+    expected_price = fields.Float(string="Expected Price", required=True)
+    selling_price = fields.Float(
+        string="Selling Price",
+        readonly=True,
+        copy=False,
+    )
```

Comment:
Dependi
```suggestion
    selling_price = fields.Float(string="Selling Price", readonly=True, copy=False)
```
Depending on the team, some prefer reducing the number of lines.
Up to you.